web/src/main/java/ua/com/vertex/dao/CourseDaoImpl.java

```diff
@@ -49,4 +82,168 @@ public CourseDaoImpl(DataSource dataSource) {
+        source.addValue(COLUMN_COURSE_NAME, course.getName());
+        source.addValue(COLUMN_COURSE_START, course.getStart());
+        source.addValue(COLUMN_COURSE_FINISHED, course.isFinished());
+        source.addValue(COLUMN_COURSE_TEACHER_NAME, course.getTeacherName());
+        source.addValue(COLUMN_COURSE_SCHEDULE, course.getSchedule());
+        source.addValue(COLUMN_COURSE_NOTES, course.getNotes());
+
+        return jdbcTemplate.update(query, source);
+    }
+
+    @Override
+    public Optional<Course> getCourseById(int courseId) throws DataAccessException {
+        LOGGER.debug(String.format("Try get course by id -(%s)", courseId));
+
+        String query = "SELECT c.id, c.name, c.start, c.finished, c.price, c.teacher_name, c.schedule, c.notes " +
+                "FROM Courses c WHERE id=:id";
+        Course course = null;
+        try {
+            course = jdbcTemplate.queryForObject(query, new MapSqlParameterSource(COLUMN_COURSE_ID, courseId),
+                    (resultSet, i) -> new Course.Builder().setId(resultSet.getInt(COLUMN_COURSE_ID))
+                            .setName(resultSet.getString(COLUMN_COURSE_NAME))
+                            .setStart(resultSet.getTimestamp(COLUMN_COURSE_START).toLocalDateTime())
+                            .setFinished((resultSet.getInt(COLUMN_COURSE_FINISHED) == 1))
+                            .setPrice(resultSet.getBigDecimal(COLUMN_COURSE_PRICE))
+                            .setTeacherName(resultSet.getString(COLUMN_COURSE_TEACHER_NAME))
+                            .setShedule(resultSet.getString(COLUMN_COURSE_SCHEDULE))
+                            .setNotes(resultSet.getString(COLUMN_COURSE_NOTES)).getInstance());
+        } catch (EmptyResultDataAccessException e) {
+            LOGGER.warn(String.format("The course with id - %s was not found.", courseId));
+        }
+
+        return Optional.ofNullable(course);
+    }
+
+    @Override
+    public List<User> getUsersAssignedToCourse(int courseId) {
+        LOGGER.debug(String.format(logInfo.getId() + " Retrieving users assigned to the course (id=%d)", courseId));
+
+        List<User> users;
+        String query = "SELECT user_id, email, first_name, last_name, phone FROM Course_users " +
+                "WHERE course_id=:courseId";
+
+        users = jdbcTemplate.query(query, new MapSqlParameterSource(COLUMN_COURSE_ID2, courseId), this::mapUser);
+
+        LOGGER.debug(logInfo.getId() + "Retrieved users with email=(" + userIdsToString(users) + ")");
+
+        return users;
+    }
+
+    @Override
+    public void removeUserFromCourse(CourseUserDto dto) {
+        LOGGER.debug(String.format(logInfo.getId() + "Removing the user id=%d from the course id=%d",
+                dto.getUserId(), dto.getCourseId()));
+
+        MapSqlParameterSource mapSqlParameterSource = new MapSqlParameterSource();
+        mapSqlParameterSource.addValue(COLUMN_COURSE_ID2, dto.getCourseId());
```

Comment:
There was already COLUMN_COURSE_ID in the scope, I had no idea how to separate these constant names correctly... Ok, I'll change it somehow...